src/gsi/fv3_regional_interface.f90

```diff
@@ -38,7 +38,7 @@ subroutine convert_fv3_regional
 !!!!!!!!!!! get grid specs !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
   grid_spec='fv3_grid_spec'            ! horizontal grid information
   ak_bk='fv3_akbk'                     ! vertical grid information
-  call bg_fv3regfilenameg%init(grid_spec_input='fv3_grid_spec',ak_bk_input='fv3_akbk')
+  call bg_fv3regfilenameg%init(6,grid_spec_input='fv3_grid_spec',ak_bk_input='fv3_akbk')
```

Comment:
could use "call bg_fv3regfilenameg%init(it=6 ..."  to make it more readable?

Reply:
Updated fv3_regional_interface.f90